app/views/teams/_business_unit_detail.html.slim

```diff
@@ -64,3 +64,10 @@ p
                 new_team_user_path(team_id: team.id, role: team.role),
                 id: 'action--new-responder-user',
                 class: 'button')
+hr
+
+- if @team.active?
+  = link_to "Deactivate team", team_path(@team.id),
```

Comment:
This should only show if the user is permitted to `destroy` the team via the `TeamPolicy`.